src/DurableTask.Netherite/Tracing/ClientTraceHelper.cs

```diff
@@ -111,11 +111,11 @@ public void TraceSend(PartitionEvent @event)
             {
                 if (this.logger.IsEnabled(LogLevel.Debug))
                 {
-                    this.logger.LogDebug("{client} Sending event {eventId}: {event}", this.tracePrefix, @event.EventIdString, @event);
+                    this.logger.LogDebug("{client} Sending event {eventId} to partition {partitionId}: {event}", this.tracePrefix, @event.EventIdString, @event.PartitionId, @event);
```

Comment:
just for my .NET knowledge: what does the `@` in `@event.PartitionId` do?

Reply:
Because `event` is a keyword in C#, you can't use it unless you prefix it `@`. You could say the '@' is an un-keyword operator.

I actually don't use this very often anymore because it is a bit distracting. I think it is usually preferrable to just use a different name.

Reply:
oh, fancy. Thanks for clarifying